src/Compute/Compute/ChangeLog.md

```diff
@@ -25,10 +25,13 @@
 * Added `Etag` property to PSVirtualMachine and PSVirtualMachineScaleSet objects.   
 * Added parameters `-IfMatch` and `-IfNoneMatch` to `Update-AzVM`, `Update-AzVmss`, `New-AzVm`, `New-AzVmss`, `New-AzVmConfig`, and `New-AzVmssConfig` cmdlets.
 * Resolved the bug with `New-AzVMConfig` for `-CommunityGalleryImageId` and `-SharedGalleryImageId` parameters.
+* [Breaking Change] Added ValidateNotNullOrEmpty for `-ResourceGroupName` and `-VMScaleSetName` parameters to `Get-AzVmss` cmdlet. [#20095]
+* [Breaking Change] Added new business logic to `New-AzVmss` and `New-AzVM` cmdlets. When the user explicitly sets the `SecurityType` to `Standard`, the Image alias defaults to `Win2022AzureEdition` to make future migrations to Trusted Launch easier.
 
 ## Version 7.3.0
 * Added cmdlet `Invoke-AzSpotPlacementRecommender`.
 * Fixed `Update-AzCapacityReservationGroup` to remove Subscriptions from SharingProfile.
+* Added new optional parameter `SecureVMGuestStateSAS` to cmdlet `Grant-AzDiskAccess`.
```

Comment:
Why to add this line in "version 7.3.0". Please refer to https://www.powershellgallery.com/packages/Az.Compute/7.3.0. Usually we do not update the former versions. It doesn't work.

Reply:
Hi @Sandido , I've merged this PR to save time. Please double confirm why to add this line in a former release. If there is a reason, please comment in this PR and let's see what's the right thing to do.

Reply:
@YanaXu , looks like that incorrectly merged. It should go in the latesdt release as that PR was merged on 4/25, https://github.com/Azure/azure-powershell/pull/24635